assets/js/blocks/featured-product/block.js

```diff
@@ -301,7 +282,10 @@ const FeaturedProduct = ( {
 		return (
 			<ResizableBox
 				className={ classes }
-				size={ { height } }
+				size={ {
+					height: '',
+					width: '',
+				} }
```

Comment:
How does that work? I seem to be able to change the height of the block, but not the width. Is that the expected behavior?

Reply:
I think that we should not limit the height of the editor.
Now the user can change the font size. If the font size is high, the block should become bigger otherwise the user can't see the entire text.

In accordance with TS type, we need to specify either `height` and `width`

Reply:
Now that the font size can't be modified anymore, should we undo this change?